<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>HeaderPanel</class>
<widget class="QWidget" name="HeaderPanel">
<property name="geometry">
<rect>
<x>0</x>
<y>0</y>
<width>480</width>
<height>73</height>
</rect>
</property>
<property name="windowTitle">
<string>Form</string>
</property>
<property name="styleSheet">
<string notr="true">background-color: rgb(0, 0, 0);</string>
</property>
<layout class="QHBoxLayout" name="horizontalLayout">
<property name="spacing">
<number>0</number>
</property>
<property name="margin">
<number>0</number>
</property>
<item>
<widget class="HeaderLeds" name="header_leds" native="true">
<property name="sizePolicy">
<sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
<horstretch>0</horstretch>
<verstretch>0</verstretch>
</sizepolicy>
</property>
<property name="minimumSize">
<size>
<width>48</width>
<height>72</height>
</size>
</property>
<property name="styleSheet">
<string notr="true">background-color: rgb(255, 85, 0);</string>
</property>
</widget>
</item>
<item>
<widget class="ATCMbutton" name="atcmButton_x">
<property name="minimumSize">
<size>
<width>72</width>
<height>73</height>
</size>
</property>
<property name="font">
<font>
<family>DejaVu Sans</family>
<pointsize>24</pointsize>
</font>
</property>
<property name="iconSize">
<size>
<width>48</width>
<height>48</height>
</size>
</property>
<property name="toolTip">
<string/>
</property>
<property name="whatsThis">
<string>ATCMbutton widget</string>
</property>
<property name="pageName">
<string notr="true"/>
</property>
<property name="statusvar">
<string notr="true"/>
</property>
<property name="viewStatus">
<bool>false</bool>
</property>
<property name="visibilityVar">
<string notr="true"/>
</property>
<property name="bgColor">
<color>
<red>0</red>
<green>0</green>
<blue>0</blue>
</color>
</property>
<property name="fontColor">
<color>
<red>255</red>
<green>85</green>
<blue>0</blue>
</color>
</property>
<property name="text">
<string>X</string>
</property>
<property name="pressText">
<string>X</string>
</property>
<property name="borderWidth">
<number>0</number>
</property>
<property name="borderRadius">
<number>0</number>
</property>
</widget>
</item>
<item>
<widget class="QPushButton" name="pushButton_time">
<property name="font">
<font>
<family>DejaVu Sans</family>
<pointsize>32</pointsize>
<weight>75</weight>
<bold>true</bold>
</font>
</property>
<property name="styleSheet">
<string notr="true">color: rgb(255, 85, 0);</string>
</property>
<property name="text">
<string>☼HH:mm</string>
</property>
</widget>
</item>
<item>
<widget class="ATCMbutton" name="atcmButton_y">
<property name="minimumSize">
<size>
<width>72</width>
<height>72</height>
</size>
</property>
<property name="font">
<font>
<family>DejaVu Sans</family>
<pointsize>24</pointsize>
</font>
</property>
<property name="toolTip">
<string/>
</property>
<property name="whatsThis">
<string>ATCMbutton widget</string>
</property>
<property name="pageName">
<string notr="true"/>
</property>
<property name="statusvar">
<string notr="true"/>
</property>
<property name="viewStatus">
<bool>false</bool>
</property>
<property name="visibilityVar">
<string notr="true"/>
</property>
<property name="bgColor">
<color>
<red>0</red>
<green>0</green>
<blue>0</blue>
</color>
</property>
<property name="fontColor">
<color>
<red>255</red>
<green>85</green>
<blue>0</blue>
</color>
</property>
<property name="text">
<string>Y</string>
</property>
<property name="pressText">
<string>Y</string>
</property>
<property name="borderWidth">
<number>0</number>
</property>
<property name="borderRadius">
<number>0</number>
</property>
</widget>
</item>
<item>
<widget class="ATCMbutton" name="atcmButton_home">
<property name="sizePolicy">
<sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
<horstretch>0</horstretch>
<verstretch>0</verstretch>
</sizepolicy>
</property>
<property name="minimumSize">
<size>
<width>50</width>
<height>50</height>
</size>
</property>
<property name="font">
<font>
<family>DejaVu Sans</family>
<pointsize>24</pointsize>
<weight>50</weight>
<bold>false</bold>
</font>
</property>
<property name="toolTip">
<string/>
</property>
<property name="whatsThis">
<string>ATCMbutton widget</string>
</property>
<property name="pageName">
<string>HOME</string>
</property>
<property name="statusvar">
<string notr="true"/>
</property>
<property name="viewStatus">
<bool>false</bool>
</property>
<property name="visibilityVar">
<string notr="true"/>
</property>
<property name="bgColor">
<color>
<red>0</red>
<green>0</green>
<blue>0</blue>
</color>
</property>
<property name="fontColor">
<color>
<red>255</red>
<green>85</green>
<blue>0</blue>
</color>
</property>
<property name="bgPressColor">
<color>
<red>255</red>
<green>255</green>
<blue>0</blue>
</color>
</property>
<property name="text">
<string>☰</string>
</property>
<property name="pressText">
<string>☰</string>
</property>
<property name="borderWidth">
<number>0</number>
</property>
<property name="borderRadius">
<number>0</number>
</property>
</widget>
</item>
</layout>
</widget>
<customwidgets>
<customwidget>
<class>ATCMbutton</class>
<extends>QPushButton</extends>
<header>atcmplugin.h</header>
</customwidget>
<customwidget>
<class>HeaderLeds</class>
<extends>QWidget</extends>
<header>headerleds.h</header>
<container>1</container>
</customwidget>
</customwidgets>
<resources/>
 <connections>
  <connection>
   <sender>atcmButton_home</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>HeaderPanel</receiver>
   <slot>goto_page(const char*,bool)</slot>
  </connection>
 </connections>
</ui>
